src/plugins/numberline/numberline-tile-component.tsx

```diff
@@ -8,25 +8,18 @@ import { useToolbarTileApi } from "../../components/tiles/hooks/use-toolbar-tile
 import { NumberlineContentModelType } from "./models/numberline-content";
 import { useCurrent } from "../../hooks/use-current";
 import { ITileExportOptions } from "../../models/tiles/tile-content-info";
-
+import { HotKeys } from "../../utilities/hot-keys";
 
 import "./numberline-tile-component.scss";
 
 export const NumberlineTileComponent: React.FC<ITileProps> = (props) => {
-  const {
-    documentContent, model, readOnly, scale, tileElt,
-    onRegisterTileApi, onUnregisterTileApi
-  } = props;
+  const { documentContent, model, readOnly, scale, tileElt,
+          onRegisterTileApi, onUnregisterTileApi } = props;
 
   const contentRef = useCurrent(model.content as NumberlineContentModelType);
-
+  const hotKeys = useRef(new HotKeys());
   const toolbarProps = useToolbarTileApi({ id: model.id, enabled: !readOnly, onRegisterTileApi, onUnregisterTileApi });
 
-  const handlePlacePoint = () => {
```

Comment:
Got rid of this for now since we are always in "Place Point" mode - may bring it back in the future if there is a change to the UI deisgn.